micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSView.js

```
@@ -124,21 +136,19 @@
-    //action===EDIT go to edit screen 
-    if(actionSelected==="EDIT") {
-      const additionalParamString = new URLSearchParams(additionalParams).toString();
-      history.push(`/${window?.contextPath}/employee/workbench/mdms-edit?moduleName=${moduleName}&masterName=${masterName}&uniqueIdentifier=${uniqueIdentifier}${additionalParamString ? "&"+additionalParamString : ""}`)
-    }
-    //action===DISABLE || ENABLE call update api and show toast respectively
-    else{
-      //call update mutation
-      handleEnableDisable(actionSelected)
-    }
-  }
+  let propsToSend = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'propsToSend' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)